model/src/wav_import_export.F90

```diff
@@ -371,6 +374,14 @@ subroutine import_fields( gcomp, time0, timen, rc )
       if (state_fldchk(importState, 'So_u')) then
         call SetGlobalInput(importState, 'So_u', vm, global_data, rc)
         if (ChkErr(rc,__LINE__,u_FILE_u)) return
+
+        if(casename == "ufs.hafs") then
```

Comment:
I think you need trims on your ``casename`` comparisons?

Reply:
@DeniseWorthen Updates have been made. Thanks!